### PR TITLE
Testing with py36

### PR DESCRIPTION
**Pull request by tk0miya**
As a trial, I added a test with py36.
At same time, I removed some test matrix patterns to save testing times.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +18,16 @@ env:
   matrix:
     - DOCUTILS=0.12
     - DOCUTILS=0.13.1
+matrix:
+  exclude:
+    - python: 3.4
```
Comment (tk0miya) [inline .travis.yml]: Hmm? It seems this does not works...why?

Comment (tk0miya) [reply .travis.yml]: Okay, I need to quote the version.

**Comment by tk0miya** [comment]
@shimizukawa Do you have any opinion? If none, I will merge this soon.

File: tox.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist=flake8,py27,py34,py35,pypy,du12,du11,du10
+envlist=flake8,py27,py34,py35,py36,pypy,du12,du11
```
Comment (shimizukawa) [inline tox.ini]: I think we need `du13`.

**Comment by shimizukawa** [comment]
LGTM with nits ;)

File: tox.ini (hunk 2 above)
Comment (tk0miya) [reply tox.ini]: Added.

**Comment by tk0miya** [comment]
Thanks!

**Pull request merged**